README.md

```diff
@@ -186,6 +187,10 @@ You can then select the type of iOS device you'd like to use as a simulator
 
 ```bash
 npm run android-live-reload
+
+# If any errors occur, set the following environment variables and try again, or run the app using Android Studio
+export JAVA_HOME=/path/to/android-studio/jbr
+export ANDROID_SDK_ROOT=/path/to/Android/Sdk
```

Comment:
This is a super helpful addition, thanks! 😊 Would you mind changing these to this syntax? I just call it "insert here" syntax haha, probably has a technical term. 
Syntax:
`export JAVA_HOME=<PATH_TO_JAVA_HOME>` 
And the same idea for the syntax of `ANDROID_SDK_ROOT` variable.

It'd also be nice to have some info on how to get the location for Java home easily, maybe something like: 
Windows: `echo %JAVA_HOME%`
Mac/Linux: `echo $JAVA_HOME`

And for `ANDROID_SDK_ROOT`, that's typically in these locations: 
Mac: `/Users/<USERNAME>/Library/Android/sdk`
Windows: `C:\Program Files\Android\sdk` OR `C:/Users/<USERNAME>/AppData/Local/Android/Sdk`

Reply:
I don't currently have a `~/Library/Android` directory - would installing the Cordova dependencies fix that?

Reply:
@yndajas 
> I don't currently have a ~/Library/Android directory - would installing the Cordova dependencies fix that?

The `~/Library/Android` path is referencing where my Android SDK is installed, it may be different for your device. I don't believe the Cordova dependencies should affect that. Instructions on [how to install Android SDK can be found here](https://developer.android.com/about/versions/14/setup-sdk).